src/app/components/crypto-asset-item/crypto-asset-item-toggle.layout.tsx

```diff
@@ -0,0 +1,57 @@
+import { useState } from 'react';
+import { useDispatch } from 'react-redux';
+
+import { Box, VStack } from 'leather-styles/jsx';
+
+import { ItemLayout, Pressable, Switch } from '@leather.io/ui';
+import { spamFilter } from '@leather.io/utils';
+
+import { useCurrentAccountIndex } from '@app/store/accounts/account';
+import { manageTokensSlice } from '@app/store/manage-tokens/manage-tokens.slice';
+
+interface CryptoAssetItemToggleLayoutProps {
+  captionLeft: string;
+  icon: React.ReactNode;
+  titleLeft: string;
+  assetId: string;
+  isCheckedByDefault?: boolean;
+}
+
+export function CryptoAssetItemToggleLayout({
+  captionLeft,
+  icon,
+  titleLeft,
+  assetId,
+  isCheckedByDefault = false,
+}: CryptoAssetItemToggleLayoutProps) {
+  const accountIndex = useCurrentAccountIndex();
+  const dispatch = useDispatch();
+
+  const [isChecked, setIsChecked] = useState(isCheckedByDefault);
+
+  function handleSelection(enabled: boolean) {
+    setIsChecked(enabled);
+    dispatch(manageTokensSlice.actions.setToken({ id: assetId, enabled, accountIndex }));
+  }
+
+  const toggle = (
```

Comment:
Is there a reason to add a `const` for **toggle** and **content**?

Maybe they can just be declared inline?